lib/components/T3CeFeloginLogin/T3CeFeloginLogin.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div class="fe-login">
+    <p>{{ $props.data.message.header }}</p>
+    <p>{{ $props.data.message.message }}</p>
+    <T3Form ref="form" :elements="elements" @submit="onSubmit" />
+  </div>
+</template>
+<script>
+import T3Form from '../../components/T3Form/T3Form'
+import baseCe from '../../mixins/component/baseCe'
+
+export default {
+  name: 'T3CeFeloginLogin',
+  components: {
+    T3Form
+  },
+  extends: baseCe,
+  props: {
+    data: {
+      type: Object,
+      required: true
+    }
+  },
+  computed: {
+    redirectUrl () {
+      const element = this.data.form.elements.filter(function (element) {
+        return element.name === 'redirect_url'
+      })
+
+      if (element.length > 0) {
+        return element?.value || ''
+      }
+
+      return ''
+    },
+    elements () {
+      return this.compileFormElements(this.data.form.elements)
+    }
+  },
+  methods: {
+    compileFormElements (elements) {
+      return elements.map((element) => {
+        return {
+          value: element.value || '',
+          type: element.type || '',
+          identifier: '', // TODO get from form element
+          label: element.label || '',
+          placeholder: element.placeholder || '',
+          required: false, // TODO: parse required form validators
+          name: element.name || '',
+          validators: element.validators
+        }
+      })
+    },
+    onSubmit ({ validator, form, formRef }) {
+      const data = new FormData(formRef)
+      data.append('responseElementId', this.id.toString())
+      data.append('responseElementRecursive', '1')
+
+      this.$typo3.api.$http
+        .$post(this.data.form.action, data)
+        .then((data) => {
```

Comment:
can we use async/await to enhance readability? I recommend using the separate method for action on success and failure - because then developers are able to override this particular method (not whole login submit) to perform custom action on success/failure. 

To get initialData I recommend setting up `redirectUrl` as the path to get data - if not available then just get current URL

Also please add validation - you can use `flags.valid`


```js
 async onSubmit ({ validator, form, formRef }) {
      const { flags } = validator
      const data = new FormData(formRef)
      data.append('responseElementId', this.id.toString())
      data.append('responseElementRecursive', '1')

      if (flags.valid) {
        try {
          const response = await this.$typo3.api.$http.$post(this.data.form.action, data)
          const { redirectUrl, status } = response.content.data
          if (status === 'success') {
            try {
              await this.$store.dispatch('getInitialData', { path: redirectUrl || this.$route.path })
            } finally {
              this.onSuccess(redirectUrl)
            }
          }
        } catch {
          this.onFailure()
        }
      }
    },
    onSuccess (redirectUrl) {
      if (redirectUrl) {
        this.$router.push(redirectUrl)
      }
    },
    onFailure () {
      alert('Login status was not success')
    }
```